### PR TITLE
Add UT coverage for multi-cloud/api pkg

### LINKED ISSUE #1083: Add UT for multi-cloud/fileshare , backend
**Issue/Feature Description:**
Add UT for multi-cloud/fileshare , backend
**Why this issue to fixed / feature is needed(give scenarios or use cases):**
To increase code reliability and code coverage 

**How to reproduce, in case of a bug:**
NA
**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
add UT coverage for multi-cloud/api pkg
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
 /kind enhancement

**What this PR does / why we need it**:
add UT coverage for multi-cloud/api pkg

**Which issue(s) this PR fixes**:

Fixes #1083 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@himanshuvar , @kumarashit Please review and merge.

File: install/CI/coverage [hunk 1]
```diff
@@ -28,6 +28,11 @@ split_line(){
 
 # Start unit test.
 split_line "Start unit test"
+# api tests
+go test -v github.com/opensds/multi-cloud/api/pkg/utils/... -cover
```
Comment (kumarashit) [inline install/CI/coverage:32]: is it possible to run the go test simply for all sub-dirs and skip any error, if test file is not present

**Review by kumarashit — APPROVED** [review]
LGTM

File: install/CI/coverage (hunk 1 above)
Comment (rajat-soda) [reply install/CI/coverage:32]: That is what 'go test' does.

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**